budget/budgetrouter.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+
+const router = express.Router();
+const Budget = require('./budget.js')
+
+router
+.route('/').post((req, res) => {
+    
+     const { body } = req;
```

Comment:
Since we're not doing anything significant with the variable `body`, we can just pass `req.body` into our `new Budget`:

```js
const budget = new Budget(req.body)
```